src/modules/LoggedIn/Project/ProjectList.tsx

```diff
@@ -1,7 +1,63 @@
 import React from 'react'
+import { Card, Row, Col } from 'antd';
+import 'antd/dist/antd.css';
+import { EllipsisOutlined } from '@ant-design/icons';
+const { Meta } = Card;
 
 const ProjectList = () => {
-  return <div>Project List</div>
+  const [card1, setCard1] = React.useState([
+    {
+      id: 1,
+      title: 'Jawa Timur Park',
+      description: 'Interactive jatim park map'
+    },
+    {
+      id: 2,
+      title: 'Museum Angkut',
+      description: 'Interactive museum angkut map'
+    },
+    {
+      id: 3,
+      title: 'Taman Safari Prigen',
+      description: 'Interactive taman safari prigen map'
+    },
+    {
+      id: 4,
+      title: 'Dino Park',
+      description: 'Interactive dino park map'
+    },
+  ])
+  const card2 = card1.map<any>(card2 => {
+    return (
+      <>
+        <Col span={6}>
+          <Card
+            style={{ width: 300 }}
+            hoverable
+            actions={[
+              <EllipsisOutlined key="ellipsis" />,
+            ]}
+          >
+            <Meta
+              title={card2.title}
+              description={card2.description}
+            />
+          </Card>
+        </Col>
+      </>
+    )
+  })
+  return (
+    <div>
+      <Row style={{ marginTop: 16 }} gutter={[24, 16]} >
+        {card2}
+      </Row>
+
+      <Row gutter={[24, 16]}>
+        {card2}
+      </Row>
```

Comment:
Bagian yang ini gak perlu dibuat dua Row, kalo Col udah dikasih props span, dia bakal otomatis bikin row baru kalo kolomnya udah lebih

```suggestion
      <Row style={{ marginTop: 16 }} gutter={[24, 16]} >
        {card2}
      </Row>
```